### PR TITLE
Backend for Graph 4: Top 3 popular Cuisine Types by Area

### PR DESCRIPTION
**Pull request by silviacasaburi**

### CODE REVIEW

**Comment by silviacasaburi** [comment]
Backend for Graph 4: Top 3 popular Cuisine Types by Area

File: backend/database/database.go [hunk 1]
```diff
@@ -1345,7 +1347,7 @@ type AreaByRating struct {
 func (d *Database) PopularVendor() ([]AreaByRating, error) {
 	const command = `
 					SELECT Vendor.Name as 'BusinessName' , AreaName as 'Location',
-					count(distinct(StarRating)) as "TotalRatings"
+					count(distinct(StarRating)) as "TotalRating"
```
Comment (MakotoE) [inline backend/database/database.go:1350]: You have a `TotalRatings` field in `AreaByRating`, so you should change the above column name to match that.

**Review by MakotoE — CHANGES_REQUESTED** [review]
The package-lock.json file is unnecessary but you added it in this PR. Please remove that file.

**Comment by silviacasaburi** [comment]
Hi Makoto, I just added  Graph 3: Top 5 Popular Searching Queries in a certain month, to the backend. Can you please review?

Thank you,
Silvia

**Review by MakotoE — CHANGES_REQUESTED** [review]
You deleted app/package-lock.json which is the one that needs to be kept.

File: backend/database/database.go [hunk 2]
```diff
@@ -1369,3 +1371,68 @@ func (d *Database) PopularVendor() ([]AreaByRating, error) {
...
+	Location    string
+	TotalRating int
+}
+
+func (d *Database) PopularCuisine() ([]CuisineByArea, error) {
+	const command = `
+SELECT AreaName as 'Location', CuisineType,count(distinct(StarRating)) as 'TotalRating'
+FROM Areas
+inner join CuisineTypes ON Areas.VendorID = CuisineTypes.VendorID
+inner join reviews ON CuisineTypes.VendorID = Reviews.VendorID
```
Comment (MakotoE) [inline backend/database/database.go:1387]: Please change `reviews` to `Reviews`.

**Review by MakotoE — CHANGES_REQUESTED** [review]

**Review by MakotoE — APPROVED** [review]

**Pull request merged**